mealprep/src/Components/Login.js

```diff
@@ -0,0 +1,50 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+
+async function loginUser(url) {
+ return fetch(url, {
+   method: 'POST',
+   headers: {
+     'Content-Type': 'application/json'
+   }
+ })
+   .then(data => data.json())
+}
+
+export default function Login({ setToken }) {
+  const [username, setUserName] = useState();
+  const [password, setPassword] = useState();
+
+  const handleSubmit = async e => {
```

Comment:
Might be useful to pass this through from Main.jsx so that we can store the username to the state and use it for queries later on

Reply:
good point, probably same goes for some of the other stuff